src/source/PeerConnection/PeerConnection.c

```diff
@@ -921,6 +921,19 @@ STATUS peerConnectionGetCurrentLocalDescription(PRtcPeerConnection pRtcPeerConne
     return retStatus;
 }
 
+/// @brief parses string of form "$number $whatever" returns $number as uint32
+/// @return 0 if value is not parsable or null
+static UINT32 parseExtId(PCHAR extmapValue)
```

Comment:
why?
it is not exposed to external compilation units

Reply:
Older C compilers won't work.

Reply:
Also, we use compiler explicit to control the compile unit export

Reply:
@MushMal We have `static` in other places in the code: https://github.com/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/blob/master/src/source/PeerConnection/Rtcp.c#L6 (+ 3 more instances in that file)

I guess we need to fix that as well?

Reply:
Oh man... Haven't seen those. Yeah, ideally we get this as low-compilance as possible. I am very concerned when language concepts try to affect the OS and loader behaviors. I would much rather keep it explicit in the compiler flags.

We really only need visibility in our case. This is controlled by PUBLIC_API macro. This allows us to control which static compile unit goes in which dynamic library. This allows us to have two dynamic libraries including the same static lib go into a single application without any collisions. I have seen projects that create such a mess with the exposures when it comes to platform level code and portability

Reply:
> Older C compilers won't work.
why do you care about old compilers?

Reply:
> > Older C compilers won't work.
> 
> why do you care about old compilers?

The codebase we use is super portable. It can be used on a number of platforms with a number of compilers in a number of environments. For example, in case of an embedded device or a kernel mode application, the compiler selection is very stringent and it would have certain linkage requirements.

Thus, we try to use the lowest common ground in order to have a single codebase. It's not always possible and in very rare cases we isolate those issues in files that have conditional compilation.